example/android/app/build.gradle

```diff
@@ -157,10 +157,10 @@ dependencies {
     implementation "com.facebook.react:react-native:+"  // From node_modules
 
     // tests
-    androidTestImplementation project(":detox")
+    androidTestImplementation ('com.wix:detox:+') { transitive = true }
     androidTestImplementation 'junit:junit:4.12'
-    androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test:rules:1.0.2'
+    // androidTestImplementation 'androidx.test:runner:1.1.1'
```

Comment:
Comment left out.
Do we need those deps?

Reply:
I think it's not needed, let me build and check in local once to confirm this

Reply:
removed comments